.github/workflows/ci.yml

```diff
@@ -26,6 +26,7 @@ jobs:
     - name: Setup cmake
       uses: jwlawson/actions-setup-cmake@v1.12
       with:
+        # CMake version 3.17 is required to build libwebp (which libsharpyuv is part of) on macOS.
```

Comment:
Does libsharpyuv.cmd build the entire libwebp? Is it possible to build just libsharpyuv?